ultravox/model/modified_whisper.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
```

Comment:
Suggest calling this file `whisper_model_modified.py` to keep the same convention as the uv files.